08-rails/active-record-migrations.md

```diff
@@ -14,42 +14,54 @@ __Question: What is the role of _models_ in the MVC pattern?__
 
 So in this lesson we will be learning how to create, change and structure the Model layer of our application.  
 
-## Generating New Rails Models & Migrations
+## Generating a New Rails Model Migration
 An important component of the `rails` command line tool is the generator. We can use the generator to automatically create idiomatic classes and files in the appropriate directories for the most common application roles.
 
-Using the generator ensures that we are following Rails _conventions_ and really speeds project set up. Let's run a generator command:
+Using the generator ensures that we are following Rails _conventions_ and really speeds project set up. Let's run a generator command to a create a new model. Note: The name we specify, "book" is _singular_ (unlike the controller command).
 
 ```bash
-$ rails generate model student name:string pie:string birthday:datetime
+$ rails generate model book title:string author:string
 ```
 
-This command will do three things:
+This command will do two things:
 
-- Create a Rails _model_ in the `app/models/` directory called `student.rb`
-- Create a Rails _migration_ in the `db/migrations/` directory called `<timestamp>_create_students.rb`
-- Put enough code in each of those files to get us going.
+- Create an empty Rails **model** in the `app/models/` directory called `book.rb`
+- Create a Rails **migration** in the `db/migrations/` directory called `<timestamp>_create_books.rb` that contains the fields specified in the command above
 
-Each file plays a role in describing how Rails interacts with the database and what information the database contains. Let's focus on _migrations_ in this discussion.
+Each file plays a role in describing how Rails interacts with the database and what information the database contains. We are  focusing on _migrations_ in this discussion. Let's open up the migration file and see what we have.
+
+Note: If you do `rails generate model <name>` _without_ any of the fields, you will still be able to specify the fields within the migration file that was created.
+
+
+### Running (and reversing) Migrations
```

Comment:
I think this section makes more sense as a subsection under "Active Record Migrations" (before "Migrations without Models".